1 table.sql

```diff
@@ -0,0 +1,20 @@
+use ArmyDB
+go
+drop table if exists dbo.Soldier
+go
+create table dbo.Soldier(
+    SoldierID int not null identity primary key,
+    FirstName varchar(25) not null constraint ck_Soldier_first_name_cannot_be_blank check(FirstName <> ''),
+    LastName varchar(25) not null constraint ck_Soldier_last_name_cannot_be_blank check(LastName <> ''),
+    SSN char(10) not null constraint ck_Soldier_SSN_cannot_be_blank check(SSN <> ''),
```

Comment:
SSN if you use `char(10)` you need to have a constraint that would make sure it's 10 characters long. And make sure they are ints.